### DISCUSSION TITLE
Timeline for next release

### DISCUSSION THREAD

**Discussion by bminixhofer** [h2oai/wave#1001] [General]
Hi! There's been lots of really useful features added since the last release, is there any approximate timeline when 0.18 will be released?

**Comment by bminixhofer** [comment]
I guess this question was answered implicitly, thanks for the release!

**Reply by mturoci** [reply]
@bminixhofer Wave now supports nightly releases so you can basically use any new feature / fix that was merged on previous day. Also feel free to report any regressions / feedback on new features in nightly. As long as the features are not officially released, we can improve the API if necessary.

Note that HAC doesn't support nightly just yet.
